lib/net.js

```diff
@@ -256,6 +258,9 @@ function Socket(options) {
 
   // Used after `.destroy()`
   this[BYTES_READ] = 0;
+
+  this[kLastTimeout] = 0;
+  this[kPrevTimeout] = 0;
```

Comment:
Can you add a comment explaining what those symbols do, and which value would they contain over time?

Can you also add a test for this beahavior in net, even if it is just internal?